### PR TITLE
.circleci/config.yml : updates rtl8730e/hello to rtl8730e/hello_8M

### PR DESCRIPTION
**Pull request by ritesh55555**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -177,6 +177,6 @@ workflows:
       - rtl8720e/loadable_apps:
           requires:
             - checkout_code
-      - rtl8730e/hello:
+      - rtl8730e/hello_8M:
           requires:
```
Comment (Tirtha-Samsung) [inline .circleci/config.yml:181]: This PR ```.circleci/config.yml : updates rtl8730e/hello to rtl8730e/hello_8M``` looks good to me

**Review by Tirtha-Samsung — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -177,6 +177,6 @@ workflows:
...
             - checkout_code
```
Comment (amogh-samsung) [inline .circleci/config.yml:182]: Looks all good,
Please do consider adding a short description to the commit though.

**Review by thapav — APPROVED** [review]
LGTM

**Review by sunghan-chang — COMMENTED** [review]
@jeongarmy added `rtl8730e/loadable_8m`. Could you add it as well?

**Comment by sunghan-chang** [comment]
@ritesh55555 I suggest commit title as `fixup! circleci: Update rtl8730e configurations`. Because this is fix of missing from previous commit.

**Comment by jeongarmy** [comment]
Could you add rtl8730e/loadable_8M for that?

**Pull request closed without merging**